packages/cli/src/commands/env/index.ts

```diff
@@ -104,13 +108,23 @@ const COMMAND_CONFIG = {
   pull: ['pull'],
 };
 
+function parseEnvironmentTarget(
+  environment = 'development'
```

Comment:
This default value seem unnecessary since this function isnt called when there is no value